docs/guides/react-components/install-plain-java.md

```diff
@@ -8,8 +8,169 @@ For this installation, you must have:
 - OpenJDK 21+ locally installed
 - Camunda `8.6.0-alpha2` or later
 
+### Download and configure Elasticsearch
+
+1. Download [Elasticsearch 8.9.2](https://www.elastic.co/downloads/past-releases/elasticsearch-8-9-2).
+2. Disable Elasticsearch's security packages by setting the `xpack.security.*` configuration options to `false` in `ELASTICSEARCH_HOME/config/elasticsearch.yml`.
+3. Start Elasticsearch by running `ELASTICSEARCH_HOME/bin/elasticsearch` (or `ELASTICSEARCH_HOME\bin\elasticsearch.bat` on Windows).
+
+### Download and configure Camunda
+
 1. Download the [latest release artifact](https://github.com/camunda/camunda/releases), starting with [8.6.0-alpha2](https://github.com/camunda/camunda/releases/tag/8.6.0-alpha2).
-2. Download [Elasticsearch 8.9.2](https://www.elastic.co/downloads/past-releases/elasticsearch-8-9-2).
-3. For non-production cases, disable Elasticsearch's security packages by setting the `xpack.security.*` configuration options to `false` in `ELASTICSEARCH_HOME/config/elasticsearch.yml`.
-4. Start Elasticsearch by running `ELASTICSEARCH_HOME/bin/elasticsearch` (or `ELASTICSEARCH_HOME\bin\elasticsearch.bat` on Windows).
-5. To start Camunda, run `bin/camunda` (or `bin\camunda.bat` on Windows).
+2. Modify the `application.yaml` to include the Elasticsearch exporter by adding the following to line 91:
+
+```
+    exporters:
+      elasticsearch:
+        className: io.camunda.zeebe.exporter.ElasticsearchExporter
+        args:
+          url: http://localhost:9200
+          index:
+            prefix: zeebe-record
+```
```

Comment:
~~~suggestion
2. Modify the `application.yaml` to include the Elasticsearch exporter by adding the following as a child of the `zeebe`/`broker` configuration element:

```
zeebe:
  broker:
    ...
    exporters:
      elasticsearch:
        className: io.camunda.zeebe.exporter.ElasticsearchExporter
        args:
          url: http://localhost:9200
          index:
            prefix: zeebe-record
```
~~~


My alpha2 download had different line numbers than yours (not sure why). Specifying the _logical_ placement of this configuration element should avoid problems with that drift, and help avoid mis-indentation and mis-placement.